src/index.ts

```diff
@@ -226,11 +230,8 @@ function b32encodeXemAddr(data: Buffer): string {
 }
 
 function b32decodeXemAddr(data: string): Buffer {
-  if(!isValid(data)) {
```

Comment:
This needs to remain in, unless you have something else to check validity.

Reply:
adding it right back